lib/graphql/fragment_cache/cache_key_builder.rb

```diff
@@ -11,10 +11,24 @@ module FragmentCache
 
     using(Module.new {
       refine Array do
+        def traverse_argument(argument)
```

Comment:
@DmitryTsepelev We should refactor `traverse_argument`, which I copied and pasted from `CacheKeyBuilder` into the `Array` refinement. I didn't see a good place to put the method where the refinement can still access it, so if you have a suggestion for a better place where both `selections_cache_key` and `path_cache_key` can access it, I'll be happy to make the change.

Reply:
I think it's fine to keep it in the refinement

Reply:
Good to go then!